src/invoices/InvoiceDetails/InvoiceDetails.css

```diff
@@ -0,0 +1,15 @@
+.InvoiceLinesLabel {
```

Comment:
please check https://github.com/folio-org/stripes-components/blob/master/docs/CSSinStripes.md
at least first letter shouldn't be in uppercase